fluid-music/src/groove.js

```diff
@@ -0,0 +1,80 @@
+const grooves = require("../grooves/grooveLibrary")
+const tab = require("./tab")
+
+const getGroove = function(groovelib, grooveName){
+
+    for(const val of Object.values(groovelib)){
+        if (val === grooveName)
+            return groovelib
+        else if (typeof val === "object"){
+            let g = getGroove(val, grooveName)
+            if (g !== null)
+                return g
+        }
+            
+    }
+    return null;
+}
```

Comment:
Same here. This is the end of a statement assigning the function to the `getGroove` variable. Even though statements don't always requires a semi colon, it's worth adding them just for consistency sake. Look through the rest of the document for missing semicolons, (ex line 33).